packages/netlify-cms-core/src/actions/config.js

```diff
@@ -41,12 +41,29 @@ export function applyDefaults(config) {
         map.set('public_folder', defaultPublicFolder);
       }
 
+      // default values for the slug config
```

Comment:
We should do this other config items instead of duplicating the defaults

Reply:
Agreed 👍